vagrant-spk

```diff
@@ -395,6 +395,86 @@ mkdir -p /var/run
 /usr/sbin/nginx -g "daemon off;"
 """
 
+DIY_SETUP_SCRIPT = r"""#!/bin/bash
+# This script is run in the VM once when you first run `vagrant-spk up`.  It is
```

Comment:
Perhaps we could write a warning, like, 

Given that `setup` is run exactly once, if you change the contents of this file, you will have to call `vagrant destroy`, or execute the script manually within the VM. If that sounds like too much work, use `build` instead.

(as a proposal)